com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -60,7 +60,9 @@ class AfterPostProcessPassData
                 parameters.blueNoise,
                 inputColor,
                 afterPostProcessBuffer,
-                depthBuffer,
+                prepassOutput.depthBuffer,
```

Comment:
Was this tested with MSAA?
I think we need prepassOutput.resolvedDepthBuffer here